### DISCUSSION TITLE
Compare stocks

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#44] [Ideas]
Google have a neat feature that allow to compare stocks toghether, we should have this too ! 

![image](https://user-images.githubusercontent.com/794449/110513237-02016500-80d4-11eb-9b62-21c0ddcc72e7.png)

**Comment by mkitzmann** [comment]
This would also be great for indexes and etfs. So if I want to decide which one of a certain kind of etf i want to buy i want to compare it to similiar ones.

I would think there should be a button `Add to comparison` on the stock/etf/index detail page. When clicked the item is added to a `compare bucket` in the header. The bucket would display the number of items as a badge. When the bucket is clicked you get to a compare page showing all selected items. There might be a limit of like 5 items, so the page doesnt get to crowded.

![stock-actions](https://user-images.githubusercontent.com/35574021/110591052-e13f1b00-8178-11eb-894b-93051f9c4fe5.png)

**Reply by sadortun** [reply]
Fyi, 95% of the time, when I mention stocks, it's in the very general term. I'll try to use `assets` instead to avoid any confusion 

---

Yes! This should also be possible to compare every kind of `asset` to any other one.

Example, if you own a rental property, you should be able to compare its value over time to other assets like stocks you may own, or the general housing market inflation in your area. 

This would give you a great indication where you should invest your money. Same idea if you want to compare two rental propriety value over time.

**Reply by mkitzmann** [reply]
Awesome idea! also yes, we should stick to `assets` ;)

This will probably come much further down the way, but where would we get data about real estate value?

**Reply by sadortun** [reply]
The user can provide its updated propriety value, otherwise we can estate growth with public housing market data. In Canada, there is data easely available that gives invlation per each major town
